lighthouse-core/config/config.js

```diff
@@ -148,17 +148,15 @@ function assertValidFlags(flags) {
   }
 }
 
-
 /**
  * @implements {LH.Config.Config}
  */
 class Config {
   /**
-   * @constructor
    * @param {LH.Config.Json=} configJSON
    * @param {LH.Flags=} flags
    */
-  constructor(configJSON, flags) {
+  static async createConfigFromJson(configJSON, flags) {
```

Comment:
Constructors can't be async, so this helper function exists now.

maybe rename to `Config.createFromJson` ?

Reply:
where are my functions sir?